.cirrus.yml

```diff
@@ -7,6 +7,7 @@ redhat_version_task:
         matrix:
           - image: fedora:32
```

Comment:
As Fedora 32 reached EOL, I think we could remove it now.

Reply:
@willianrampazzo  indeed! Exactly what month ago:
https://fedoraproject.org/wiki/End_of_life
I'm going to do a force-push and remove it